addons/point_of_sale/static/src/xml/pos.xml

```diff
@@ -409,6 +409,17 @@
                             </t>
                         </select>
                     </div>
+                </div>
+                 <div class='client-detail'>
+                    <span class='label'>Language</span>
+                    <select class='detail client-lang needsclick' name='lang'>
+                        <option value=''>None</option>
```

Comment:
I'm not sure that we should set a 'None' option, could you check if you have a lang in 'session.user_context' and set this one as default?

Because the lang in the context is supposed to be the one of the user that opened the POS.